modules/bullet/bullet_physics_server.cpp

```diff
@@ -837,8 +844,12 @@ void BulletPhysicsServer3D::body_set_ray_pickable(RID p_body, bool p_enable) {
 }
 
 PhysicsDirectBodyState3D *BulletPhysicsServer3D::body_get_direct_state(RID p_body) {
+	ERR_FAIL_COND_V_MSG((using_threads && !doing_sync), nullptr, "Body state is inaccessible right now, wait for iteration or physics process notification.");
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(using_threads && !doing_sync, nullptr, "Body state is inaccessible right now, wait for iteration or physics process notification.");
```

Reply:
@sboron Please don't mark conversations as resolved without making the changes that were asked for. I've unresolved several ones.

Reply:
That was just a commit issue. Was taking the wrong one.